sites/fast-website/src/app/index.ts

```diff
@@ -1,4 +1,13 @@
+import { DesignSystem } from "@microsoft/fast-foundation";
+import * as components from "@microsoft/fast-components/dist/esm/custom-elements";
+import { siteColorSwatch } from "./components";
 import "./css/style.css";
+DesignSystem.getOrCreate().register(
+    ...Object.values(components).map(x => (x as () => void)()),
```

Comment:
I wonder if we should have a special export from custom-elements that implements Registry so that we could pass it directly to the register method without the odd syntax. Is it worth doing that for convenience?

Reply:
Oops forgot to update this, good catch. I was going to only register what was being used in the site.

Good question. It could be, maybe a `customElements` or `components` named export that is the entire component collection?

Reply:
What about something called `allComponents`?

Reply:
We could follow the same pattern in fluent.